lms/static/sass/views/_homepage.scss

```diff
@@ -0,0 +1,168 @@
+// lms - views - homepage view
+// ====================
+
+$course-card-height: ($baseline*18);
+$course-image-height: ($baseline*8);
+$course-info-height: ($baseline*10);
+$course-title-height: ($baseline*3.6);
+$learn-more-horizontal-position: calc(50% - 100px); // calculate the left position for "LEARN MORE" content
+
+.courses-container {
+  @include outer-container;
+
+  .courses {
+    @include clearfix();
+    padding: ($baseline*2) 0 ($baseline*0.75) ;
+
+    .courses-listing {
+      margin: 0;
```

Comment:
Please use the ``%ui-no-list`` extend here (https://github.com/edx/edx-platform/blob/master/lms%2Fstatic%2Fsass%2Fbase%2F_mixins.scss#L133).